python/lsst/ts/observatory/control/remote_group.py

```diff
@@ -543,9 +715,15 @@ async def standby(self):
 
     @property
     def components(self):
-        """ List of components.
+        """ List of components names.
```

Comment:
I suggest expanding this a bit and including an example. Also the same for `components_attr` below. That will make it easier to disambiguate the two.

You have all the necessary information above. I suggest you find and copy one of the long descriptions and then shrink the copies to point to this property/attribute and an example.